src/main/webapp/intro.html

```diff
@@ -212,23 +200,6 @@ <h4 class="card-title">Rick Ridley</h4>
                   <div class="card-body text-center mt-4">
                     <h4 class="card-title">Rick Ridley</h4>
                     <p class="card-text">This is a introduction part for Rick for write blablablablabla.....blablablablablabla. IAmAPlaceholder IAmAPlaceholder. IAmAPlaceholder IAmAPlaceholder IAmAPlaceholder. IAmAPlaceholder IAmAPlaceholder IAmAPlaceholder.</p>
```

Comment:
Google Software Engineer

Rick is a software engineer at Google in his third year on the RCS team.  He graduated from Stanford in 2017 with a Bachelors of Science in Computer Science and a Minor in Economics.

(You can include my Google email: rpridley@google.com)